### ISSUE TITLE
[Suggestion] Components dans l'écran d'alchimie directement

### ISSUE THREAD

**Issue by Simon-Lessard** [JulienNigon/SpellcasterUniversity-Bug-Report#887]
Une fois assez avancé dans la partie le nombre de cartes de composants augmente énormément. Ça rend l'interface plus difficil à utiliser, surtout lorsqu'il y a plus d'une page de carte. Je pense qu'il serait plus sympa de mettre les composant directement dans l'écran d'alchimie sans utiliser de carte. Ça faciliterait grandement la création des potions tout en polissant l'interface des cartes.

![too-many-cards](https://user-images.githubusercontent.com/23580881/75992546-69e90280-5ef8-11ea-959d-e9b05e2d0698.png)

Les ingrédients disponibles pourraient apparaître ici plutôt.
![alchimie](https://user-images.githubusercontent.com/23580881/75992636-8ab15800-5ef8-11ea-9c71-fa79a3bdc963.png)

**Comment by MlleMau** [comment]
Bonjour Simon-Lessard,

Le problème a déjà été soulevé, notamment ici -> https://github.com/JulienNigon/SpellcasterUniversity-Bug-Report/issues/785 L'équipe travaille déjà dessus !

**Comment by Simon-Lessard** [comment]
Super. J'essaie de troujour vérifier les doublons, mais je me base sur les suggestions de doublon de Github qui n'avait pas identifié ce ticket pour ce point.

**Comment by MlleMau** [comment]
Oui, le système de Github est vraiment... hmm, disons que j'ai très peu de considérations pour ! xD

**Comment by Neelan22** [comment]
Ma proposition au dev est de mettre les cartes directement dans le chaudron, et qu'elles aillent se mettre automatiquement dans leur emplacement. 
Il y a des progrès à faire sur cette interface, mais c'est pas simple à modifier. Ce n'est toutefois pas dans nos priorités.

**Comment by Florane88** [comment]
> Ma proposition au dev est de mettre les cartes directement dans le chaudron, et qu'elles aillent se 

Oui, j'avais vu passer cette idée, j'aime beaucoup !
